pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -199,16 +202,18 @@ func (o *LoginOptions) GatherProjectInfo() error {
 
 	switch len(projectsItems) {
 	case 0:
-		me, err := o.Whoami()
-		if err != nil {
-			return err
-		}
 		// TODO most users will not be allowed to run the suggested commands below, so we should check it and/or
 		// have a server endpoint that allows an admin to describe to users how to request projects
-		fmt.Printf(`You don't have any project.
-If you have access to create a new project, run 'openshift ex new-project <projectname> --admin=%s'.
-To be added as an admin to an existing project, run 'openshift ex policy add-user admin %s -n <projectname>'.
-`, me.Name, me.Name)
+		fmt.Printf(`
+You don't have any projects. If you have access to create a new project, run
+
+    $ openshift ex new-project <projectname> --admin=%q
```

Comment:
BTW we need a definition about tabbing size when printing cmd usage. I'm pretty sure there are different tab sizes in other places (probably including help).